docs/integration/apache.md

```diff
@@ -1,12 +1,24 @@
 title: Apache Web Server Monitoring Integration
 description: Apache monitoring integration sends workers, scoreboard, php fpm, and httpd server performance metrics and other inventory data from your OS and web server to Sematext in cloud SaaS / private on-premises infrastructure and application monitoring & log management service 
 
-## Integration
+Sematext offers a simple, easy to install, Apache monitoring agent with minimal CPU and memory overhead.
 
-- Agent: [https://github.com/sematext/sematext-agent-httpd](https://github.com/sematext/sematext-agent-httpd)
-- Instructions: [https://apps.sematext.com/ui/howto/Apache/overview](https://apps.sematext.com/ui/howto/Apache/overview)
+## Install Apache Monitoring Agent
 
-## Metrics
+Setting up the monitoring agent takes less than 5 minutes:
+
+1.  Activate Apache `mod_status` module with `sudo a2enmod status` and configure it in `status.conf` file, e.g. in `/etc/apache2/mods-enabled/status.conf`:
+
+        ExtendedStatus On
+        <Location /server-status>
+          SetHandler server-status
+        </Location>
+
+2.  Create an Apache App in the  [Integrations / Overview](https://apps.sematext.com/ui/monitoring-create) (or  [Sematext Cloud Europe](https://apps.eu.sematext.com/ui/monitoring-create)). This will let you install the agent and control access to your monitoring and logs data. The short  [What is an App in Sematext Cloud](https://www.youtube.com/watch?v=tr_qxdr8dvk&index=14&list=plt_fd32ofypflbfzz_hiafnqjdltth1ns) video has more details.
+3.  Name your Apache monitoring App and, if you want to collect Apache logs as well, create a Logs App along the way.
+4.  Install the Sematext Agent according to the [https://apps.sematext.com/ui/howto/Apache/overview](https://apps.sematext.com/ui/howto/Apache/overview) displayed in the UI.
+
```

Comment:
I think we should show the instructions here instead of linking to another page.

If we have that content, I believe it would be easier for the user to follow on this page, instead of sending them somewhere else.

Reply:
I would really like to avoid that, it would be yet another place we have to maintain. Also, these docs are not meant to be really used to install anything, but give a general feel of what is needed. Without the right region, right tokens and so on, user can't possibly install anything by using just the docs pages.